tools/diagnostics/nsdiff

```diff
@@ -439,10 +618,11 @@ class NodeStateComparator():
                         copyfile(join(abspath(dirName), fname), join(sout,
                             fname))
 
+
         # Check for a log file
         (dirname, filename, name, extension) = self.split_name(sin)
         log_filename = "{}.log".format(filename)
-        log_filepath = "{}/{}".format(dirname, log_filename)
+        log_filepath = join(dirname, log_filename)
```

Comment:
Better use `os.path.join` since `join` is builtin so you are not shadowing a buitin